build.gradle

```diff
@@ -292,45 +292,6 @@ project(":functionaltests") {
     }
 }
 
-// Publish build results to github pages settings:
```

Comment:
moved to separate script [`git-publish.gradle`](https://github.com/commercetools/commercetools-payone-integration/pull/190/files#diff-68c0c3877fea5243aa757322fc163eb8)